packages/sit-onyx/src/styles/variables-onyx.css

```diff
@@ -0,0 +1,164 @@
+  --onyx-color-brand-neutral-700: #2a3c4a;
+  --onyx-color-brand-neutral-800: #223340;
+  --onyx-color-brand-neutral-900: #192a37;
+  --onyx-color-brand-neutral-1000: #11212d;
+  --onyx-color-brand-neutral-1100: #081723;
+  --onyx-color-brand-neutral-1200: #000e19;
+  --onyx-color-universal-grey-white: #ffffff;
+  --onyx-color-universal-grey-100: #f4f4f4;
+  --onyx-color-universal-grey-200: #e0e0e0;
+  --onyx-color-universal-grey-300: #cccccc;
+  --onyx-color-universal-grey-400: #b8b8b8;
+  --onyx-color-universal-grey-500: #a4a4a4;
+  --onyx-color-universal-grey-600: #909090;
+  --onyx-color-universal-grey-700: #7d7d7d;
+  --onyx-color-universal-grey-800: #696969;
+  --onyx-color-universal-grey-900: #555555;
+  --onyx-color-universal-grey-1000: #414141;
+  --onyx-color-universal-grey-1100: #2d2d2d;
+  --onyx-color-universal-grey-1200: #191919;
+  --onyx-color-universal-grey-black: #000000;
+  --onyx-color-universal-red-100: #fbefee;
+  --onyx-color-universal-red-200: #f1c0be;
+  --onyx-color-universal-red-300: #e6918f;
+  --onyx-color-universal-red-400: #dc625f;
+  --onyx-color-universal-red-500: #d1332f;
+  --onyx-color-universal-red-600: #bc2d2a;
+  --onyx-color-universal-red-700: #a82824;
+  --onyx-color-universal-red-800: #93221f;
+  --onyx-color-universal-red-900: #7e1d19;
+  --onyx-color-universal-red-1000: #691714;
+  --onyx-color-universal-red-1100: #400c09;
+  --onyx-color-universal-red-1200: #400c09;
+  --onyx-color-universal-orange-100: #faf6f2;
+  --onyx-color-universal-orange-200: #f8dfc8;
+  --onyx-color-universal-orange-300: #f6c89e;
+  --onyx-color-universal-orange-400: #f4b074;
+  --onyx-color-universal-orange-500: #f2994a;
+  --onyx-color-universal-orange-600: #d98841;
+  --onyx-color-universal-orange-700: #bf7737;
+  --onyx-color-universal-orange-800: #a6662e;
+  --onyx-color-universal-orange-900: #8c5625;
+  --onyx-color-universal-orange-1000: #73451c;
+  --onyx-color-universal-orange-1100: #593412;
+  --onyx-color-universal-orange-1200: #402309;
+  --onyx-color-universal-green-100: #ecf8f2;
+  --onyx-color-universal-green-200: #c2e4d3;
+  --onyx-color-universal-green-300: #98d1b3;
+  --onyx-color-universal-green-400: #6ebe94;
+  --onyx-color-universal-green-500: #44aa75;
+  --onyx-color-universal-green-600: #3b9b69;
+  --onyx-color-universal-green-700: #328c5e;
+  --onyx-color-universal-green-800: #297d52;
+  --onyx-color-universal-green-900: #216d46;
+  --onyx-color-universal-green-1000: #185e3a;
+  --onyx-color-universal-green-1100: #064023;
+  --onyx-color-universal-green-1200: #064023;
+  --onyx-color-universal-purple-100: #f4f1f8;
+  --onyx-color-universal-purple-200: #e6dcf2;
+  --onyx-color-universal-purple-300: #cab4e5;
+  --onyx-color-universal-purple-400: #aa89d1;
+  --onyx-color-universal-purple-500: #9068c0;
+  --onyx-color-universal-purple-600: #7b52ab;
+  --onyx-color-universal-purple-700: #613990;
+  --onyx-color-universal-purple-800: #3a1664;
+  --onyx-number-spacing-050: 0;
```

Comment:
@jannick-ux Is it best practice to name it `050` or just `50`?

Reply:
both is fine for me. I just wanted to keep it clean with tree digits. 
But I can change it to double digit. No problem.

Reply:
I'd prefer two digits but thats just my personal opinion 😄

Reply:
50 done